bin/run-prod.sh

```diff
@@ -3,4 +3,5 @@
 python manage.py migrate
 python manage.py collectstatic --noinput
 python manage.py compress --engine jinja2 --extension=.jinja
-gunicorn moderator.wsgi:application -b 0.0.0.0:${PORT:-8000} --log-file -
+ln -s wsgi/playdoh.wsgi remo/wsgi.py
```

Comment:
Instead of linking every time we run docker prod just symlink the file in the repository.